pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -249,6 +251,10 @@ func Pred(ir *codegen.PredictIR) (string, error) {
 		Y:                 ir.TrainIR.Label.GetFieldMeta()[0],
 		ModelParams:       modelParams,
 		Save:              "model_save",
+		HDFSNameNodeAddr:  session.HdfsNamenodeAddr,
```

Comment:
Seems no test can ensure that these fields are filled?

Reply:
Using `regex` to ensure the `HDFS...` field.